internal/controllers/topology/cluster/structuredmerge/filterintent_test.go

```diff
@@ -167,6 +167,27 @@ func Test_filterIgnoredPaths(t *testing.T) {
 				// we are filtering out spec.foo and then spec given that it is an empty map
 			},
 		},
+		{
+			name: "Cleanup emptynested  maps",
```

Comment:
```suggestion
			name: "Cleanup empty nested maps",
```
nit: move the whitespace one word to the left :)